Algoritmos-geneticos/content/Algoritmos_geneticos.md

```diff
@@ -145,17 +145,17 @@ Se uma solução ótima ou satisfatória for encontrada (isto é, uma solução
 
 ---
 
-Agora você tem todas as peças de que precisa para montar seu quebra-cabeça. Abaixo você pode conferir na estrutura desse repositório vários exemplos de aplicação de algoritmos genéticos. Acesse esse [link](https://github.com/ITA-LOW/Primeiros-passos-em-algoritmos-geneticos/tree/template/code) para ir diretamente para a pasta "code"
+Agora você tem todas as peças de que precisa para montar seu quebra-cabeça. Abaixo você pode conferir na estrutura desse repositório vários exemplos de aplicação de algoritmos genéticos. Acesse esse [link](./code/acertando_uma_string.md) para ir diretamente para a pasta "code"
```

Comment:
Como vc mudou o nome do arquivo "Acertando_uma_string.md" com letra maiúscula no início, tem que passar isso pra cá tb. Então coloca:
```suggestion
Agora você tem todas as peças de que precisa para montar seu quebra-cabeça. Abaixo você pode conferir na estrutura desse repositório vários exemplos de aplicação de algoritmos genéticos. Acesse esse [link](./code/Acertando_uma_string.md) para ir diretamente para a pasta "code"
```